test/framework/integration_framework/fake_peer/network/yac_network_notifier.cpp

```diff
@@ -17,8 +17,7 @@ namespace integration_framework {
       votes_subject_.get_subscriber().on_next(state_ptr);
     }
 
-    rxcpp::observable<YacNetworkNotifier::StateMessagePtr>
-    YacNetworkNotifier::get_observable() {
+    rxcpp::observable<YacMessagePtr> YacNetworkNotifier::get_observable() {
```

Comment:
```suggestion
    rxcpp::observable<YacMessagePtr> YacNetworkNotifier::getObservable() {
```
Method name does not match the [style guide](https://google.github.io/styleguide/cppguide.html#Function_Names).